spec/features/editing_content_spec.rb

```diff
@@ -56,7 +56,7 @@
     let!(:node1) { Fabricate(:general_content, state: 'published',
       parent: section1.home_node, section: section1) }
     let!(:node2) { Fabricate(:news_article, state: 'published',
-      parent: section2.home_node, section: section2) }
+      section: section2, release_date: Date.today) }
```

Comment:
`Date.today` is the default for `:release_date` and it does not appear to be explicitly referenced. Do you need to set it?